test/system/700-play.bats

```diff
@@ -356,3 +356,12 @@ status: {}
     run_podman stop -a -t 0
     run_podman pod rm -t 0 -f test_pod
 }
+
+@test "podman kube down" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
+
+    run_podman kube play $PODMAN_TMPDIR/test.yaml
+    run_podman kube down $PODMAN_TMPDIR/test.yaml
```

Comment:
Could you add some actual test here? Merely going with "exit status is ok" is dangerous. Like, maybe a `podman inspect` for container stopped state. And please clean up too (`run_podman rm`)